GIT_AND_GITHUB.md

```diff
@@ -47,10 +47,18 @@ Use `--no-ff` when you merge to `master`.
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
+Besides a clear description more information could be helpful, for example:
+
+- If is static UI, a screenshot would do. 
+- If is an interaction a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
+- If is a bug, steps to reproduce are useful to understand the context.
+
 #### Who merges the pull request?
 
```

Comment:
I have tried both approaches and I think having the reviewer merge works better. Is always one step less, which means PRs go faster.

**When the reviewer merges**:

Case A:
1.- You submit a PR
2.- The reviewers checks and merges

Case B:
1.- You submit a PR
2.- The reviewer makes 3 comments
3.- 2 comments get fixed, one gets clarified
4.- Reviewer merges

Case C:
1.- You submit a PR
2.- The reviewer makes 3 comments
3.- 2 comments get fixed, one gets clarified
4.- Reviewer reviews, finds a problem
5.- You fix the problem
6.- Reviewer merges

**When you merge your own PRs**:

Case A:
1.- You submit a PR
2.- The reviewers checks and notifies you that is ready to merge
3.- You merge your PR

Case B:
1.- You submit a PR
2.- The reviewer makes 3 comments
3.- 2 comments get fixed, one gets clarified
4.- Reviewer notifies that is good
5.- You merge

Case C:
1.- You submit a PR
2.- The reviewer makes 3 comments
3.- 2 comments get fixed, one gets clarified
4.- Reviewer reviews, finds a problem
5.- You fix the problem
6.- Reviewer verifies fix
7.- You merge